opwen_email_client/util/network.py

```diff
@@ -0,0 +1,12 @@
+from socket import create_connection
+from socket import gethostbyname
+
+
+def check_connection(hostname, port):
```

Comment:
Please add type annotations so that auto-complete and mypy can work better with this function:

```suggestion
def check_connection(hostname: str, port: int) -> bool:
```